resources/content/schema/token/token-auth.json

```diff
@@ -2,8 +2,8 @@
     "authorize" : {
         ".*" : "",
         "schema.*": "r",
-        "token": "cr",
-        "token.accountId": "r",
+        "token": "crd",
+        "token.id": "r",
```

Comment:
@cjellick I added this id only because DELETE would call getById first, and in it's absence I would get a `MethodNotAllowed`. So if we switch to some constant string then yes I can remove this.